build.sh

```diff
@@ -7,24 +7,43 @@ tempdir=$(dirname $(mktemp -u))
 
 wintun_version="wintun-0.11"
 
-
 # until https://github.com/golang/go/issues/37475 is implemented
 VERSION=$(git describe --tags --always --abbrev=8 --dirty)
 
-
 # download dependencies
 download-wintun() {
   echo "check dependencies"
+
+  goos="$(go env GOOS)"
+  local force_download="$1"
+  if [ "$force_download" != "true" ] && [ "$goos" != "windows" ]; then
+    echo "dependencies loaded successfully"
+    return
+  fi
+
   if [[ ! -e "$tempdir/$wintun_version" ]]; then
-    if ! type "wget" > /dev/null; then
-        echo "wget util could not be found. Please install it"
-        exit
+    download_url="https://www.wintun.net/builds/$wintun_version.zip"
+    if type "wget" >/dev/null; then
+      wget "$download_url"
+    elif type "curl" >/dev/null; then
+      curl -sSL "$download_url" >"$wintun_version.zip"
+    else
+      echo "wget or curl could not be found. Please install it"
+      exit 1
     fi
-    wget "https://www.wintun.net/builds/$wintun_version.zip"
+
     unzip "$wintun_version.zip" -d "$tempdir/$wintun_version"
     rm -f "$wintun_version.zip"
   fi
-  echo "dependencies are loaded successfully"
+
+  arch="$(go env GOARCH)"
+  wintunarch="$arch"
+  if [ "$arch" == "386" ]; then
+    wintunarch="x86"
+  fi
+  cp "$tempdir/$wintun_version/wintun/bin/$wintunarch/wintun.dll" "$awldir/embeds/wintun.dll"
```

Comment:
MB we should do  it only if some sort flag  (install Dependencies) or smth like that set? Only reason is to prevent unnecessary copying, for complicated func that do it by themself (gobuild-windows)

Reply:
Done. Also, found that we didn't "install" wintun before in `./build.sh awl-tray`, now fixed.